project/Build.scala

```diff
@@ -81,7 +81,13 @@ object Shading extends Build {
       ShadeRule.rename("com.google.protobuf.**" -> "org.tensorframes.protobuf3shade.@1").inAll,
       ShadeRule.rename("google.protobuf.**" -> "org.tensorframes.google.protobuf3shade.@1").inAll
     ),
-    assemblyOption in assembly := (assemblyOption in assembly).value.copy(includeScala = false)
+    assemblyOption in assembly := (assemblyOption in assembly).value.copy(includeScala = false),
+    assemblyMergeStrategy in assembly := {
+        case PathList("org", "tensorflow", xs @ _*) => MergeStrategy.first
```

Comment:
Please add an inline comment. This happens to fix the issue, but it is not guaranteed by contract. If we can explicitly remove the class from libtensorflow_jni jar, that is safer (not needed for now).